### ISSUE TITLE
Chat is not working: completion operation does not work with the specified model

### ISSUE THREAD

**Issue by zntr1** [Azure-Samples/azure-search-openai-demo-csharp#158]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> deploy the app using azd in azure region francecentral

### Any log messages given by the failure
> Error: The completion operation does not work with the specified model, gpt-35-turbo. Please choose different model and try again. You can learn more about which models can be used with each operation here: https://go.microsoft.com/fwlink/?linkid=2197993. Status: 400 (Bad Request) ErrorCode: OperationNotSupported Content: {"error":{"code":"OperationNotSupported","message":"The completion operation does not work with the specified model, gpt-35-turbo. Please choose different model and try again. You can learn more about which models can be used with each operation here: https://go.microsoft.com/fwlink/?linkid=2197993."

### Expected/desired behavior
> After freshly deploying the app, the chat widget should make a conversation possible.

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### Versions
>

### Mention any other details that might be useful
> The chat widget cannot be used to directly chat with gpt-35-turbo because it throws the error above.. when instead chatting via the "voice" widget, it works as intented.
> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by KijoKim76** [comment]
Same here.

**Comment by emichaelsomm** [comment]
Same

**Comment by zntr1** [comment]
> Latest model change to 0613 made this issue. You can change it back to 0310.
> 
> Related change link. [eacfda5...0fc53c3](https://github.com/Azure-Samples/azure-search-openai-demo-csharp/compare/eacfda5adc3b545507a5f3aac3ecd27756d6703e...0fc53c3b8df2608c70d0ed3dedc60f99be550962)

Yeah, sadly the newer version was the whole reason to update right now. Thanks though, please do note, that its version "0301" and not "0310", otherwise you will get an error trying to revert back to the older version.

**Comment by geffzhang** [comment]
same issue
